projects/plugins/jetpack/modules/copy-post.php

```diff
@@ -317,10 +317,12 @@ public function add_row_action( $actions, $post ) {
 		);
 		$edit_action = array(
 			'jetpack-copy' => sprintf(
-				'<a href="%s" aria-label="%s">%s</a>',
+				'<a href="%s" aria-label="%s">%s <img src="%s" alt="%s" width="16" height="16" style="vertical-align:text-bottom"></a>',
 				esc_url( $edit_url ),
-				esc_attr__( 'Copy this post.', 'jetpack' ),
-				esc_html__( 'Copy', 'jetpack' )
+				esc_attr__( 'Copy this post with Jetpack', 'jetpack' ),
+				esc_html__( 'Copy', 'jetpack' ),
+				plugins_url( '', JETPACK__PLUGIN_FILE ) . '/images/jetpack-logomark-green.svg',
```

Comment:
Could we re-use one of the logos that we already ship with the Jetpack plugin? The Jetpack Logo package comes to mind:
https://github.com/Automattic/jetpack/blob/b01b38e4dd7ebafbf396d8a1136e34bdc1e68bca/projects/packages/logo/README.md#L1-L3

Reply:
Done, thanks! I figured there'd be a package I could use, but I didn't know which one.